core/src/main/scala/kafka/utils/ReplicationUtils.scala

```diff
@@ -27,7 +27,7 @@ import scala.collection._
 
 object ReplicationUtils extends Logging {
 
-  val IsrChangeNotificationPrefix = "isr_change_"
+  private val IsrChangeNotificationPrefix = "isr_change_"
 
```

Comment:
I thought that you also wanted to move the code that creates the isr_change path in ZK to the beginning of broker startup (currently it's in the controller startup)?